demos/android-supabase-todolist/local.properties.example

```diff
@@ -11,6 +11,7 @@ sdk.dir=/Users/dominic/Library/Android/sdk
 
 SUPABASE_URL=https://foo.supabase.co
 SUPABASE_ANON_KEY=foo
+SUPABASE_STORAGE_BUCKET=media # optional attachment bucket, set to null to disable
```

Comment:
IIRC our guides don't mention this bucket, so I wonder if we should set it to null in the template and then make this an opt-in feature? That's the way things are on our other SDKs, and it ensures running the example works even without the bucket being set up.

Reply:
Good point. I've updated the example default to `null`